README.md

```diff
@@ -6,13 +6,25 @@ This operator is installed by the [cluster-storage-operator](https://github.com/
 
 # Quick start
 
+Before running the operator manually, you `must` remove the operator installed by CSO/CVO, as follows :-
+
+```shell
+#Scale down CVO and CSO
+oc scale --replicas=0 deploy/cluster-version-operator -n openshift-cluster-version
+oc scale --replicas=0 deploy/cluster-storage-operator -n openshift-cluster-storage-operator
+
+#Delete operator resources (daemonset, deployments)
+oc -n openshift-cluster-csi-drivers delete deployment.apps/aws-ebs-csi-driver-operator deployment.apps/aws-ebs-csi-driver-controller daemonset.apps/aws-ebs-csi-driver-node
+```
+
 To build and run the operator locally:
 
 ```shell
 # Create only the resources the operator needs to run via CLI
-oc apply -f manifests/00_crd.yaml
-oc apply -f manifests/01_namespace.yaml
-oc apply -f manifests/08_cr.yaml
+# oc apply -f manifests/00_crd.yaml
+# oc apply -f manifests/01_namespace.yaml
+# oc apply -f manifests/08_cr.yaml
```

Comment:
`oc apply -f https://raw.githubusercontent.com/openshift/cluster-storage-operator/master/assets/csidriveroperators/aws-ebs/10_cr.yaml`

But the link may change in the future...

Reply:
yes that's why I got confused, @bertinatto  any suggestions ?

Reply:
I'm not sure if I understand the question, but I think the command above is OK for now.

Reply:
So I have removed all the manifests folder, now shall I just markdown the link `https://raw.githubusercontent.com/openshift/cluster-storage-operator/master/assets/csidriveroperators/aws-ebs` 
or refer to specific `yaml` or ?